priv/repo/migrations/20170317153551_create_message.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.Repo.Migrations.CreateMessage do
+  use Ecto.Migration
+
+  def change do
+    create table(:messages) do
+      add :body, :string, null: false
+      add :name, :string
+      add :user_id, references(:users, on_delete: :nothing), null: false
+
```

Comment:
Nah, let's keep this PR as it is, quite big already. It's only a spike atm, we can discuss how we want to model things properly later.